gensim/scripts/segment_wiki.py

```diff
@@ -63,7 +63,7 @@
 from functools import partial
 
 from gensim.corpora.wikicorpus import IGNORED_NAMESPACES, WikiCorpus, filter_wiki, find_interlinks, get_namespace, utils

```

Comment:
@mpenkov here there is already import of other `utils`. I'll use `from gensim.utils import open` here - is that ok?

Reply:
Let's use namespaces:

```
import gensim.utils
with gensim.utils.open(...) as fin:
    ...
```

Reply:
done